conf/modules/ins_vectornav.xml

```diff
@@ -48,4 +48,10 @@
       endif
     </raw>
   </makefile>
+  
+  <!-- For some reason this doesn't work ?
```

Comment:
Of course not: This will load the module (and hence try to call the init and event functions which don't exist in the NPS implementation).